### DISCUSSION TITLE
Select doesnt work in Modal

### DISCUSSION THREAD

**Discussion by pabriciosam** [gluestack/gluestack-ui#2347] [Q&A]
I'm using a Select component inside a Modal component of react-native. The Select show in screem, but when I click on, the itens doesnt shows. Someone knows how can I solve this problem?

**Comment by rvanbaalen** [comment]
Is this project even alive? This has been an issue for almost two years and is unanswered.

**Comment by Sanchitv3** [comment]
Hey! This might have been missed earlier, thanks for bringing it up 🙏

We’ll take a look into the Select inside Modal issue. It would really help us track and prioritize this properly if you could raise it as an issue instead of a discussion.

Also, could you share a minimal reproducible example? That would help us debug and resolve this much faster.
